### DISCUSSION TITLE
subscription plans

### DISCUSSION THREAD

**Discussion by edoconte** [mynaparrot/plugNmeet-server#49] [General]
where can i find the subscription plans for the platform service?

**Comment by jibon57** [comment]
plugNmeet is an opensource project. You're free to install your own server. If you find it difficult then we can help you. Please send email to bob[@]mynaparrot.com

**Comment by wbobteng** [comment]
Hello @edoconte , please contact me via email, bob[@]mynaparrot. com, I will assist you to use this software and also can discuss a subscription plan based on your requirements. Thanks
